### PR TITLE
Add docker-compose setup

### PR DESCRIPTION
**Pull request by frankwiles**
Working / foolproof docker-compose setup for local development.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  web:
+    build:
+      context: .
+    volumes:
+      - .:/code
+    ports:
+      - "8100:8000"
+    command: ["/bin/bash", "/code/start.sh"]
+    environment:
+      - DATABASE_URL=postgres://postgres@db/postgres
```
Comment (fcurella) [inline docker-compose.yml]: isn't the database called `simpl` from the `POSTGRES_DB` env var in the `db` service?

Comment (frankwiles) [reply docker-compose.yml]: Good catch!

Comment (janeeisenstein) [reply docker-compose.yml]: @frankwiles If this was a good catch, why didn't anything change?

Comment (janeeisenstein) [reply docker-compose.yml]: @frankwiles Did you change anything?

**Comment by janeeisenstein** [comment]
Shouldn't you update the README with instructions for using this?

**Comment by janeeisenstein** [comment]
@frankwiles A number of tweak branches that were created the week of the Bootcamp. Shouldn't this PR be updated to reflect them?

**Comment by janeeisenstein** [comment]
Closing this due to obsolescence.

**Pull request closed without merging**